src/edu/mills/cs180a/StringUtilitiesTester.java

```diff
@@ -14,11 +19,12 @@ void isSubstringThrowsExceptionForNullArg() {
 
     @Test
     void isSubstringReturnsTrueForEmptySubstring() {
-        assertTrue(StringUtilities.isSubstring("", "foo"));
+        assertFalse(StringUtilities.isSubstring(" ", "foo"));
```

Comment:
Oops, it looks like you added a space in the substring. You should revert this line to the original version.